apps/myaccount/src/extensions/configs/models/index.ts

```diff
@@ -17,3 +17,4 @@
  */
 
 export * from "./common";
+export * from "./account-recovery";
```

Comment:
> Add newline

Reply:
A new line is already added here. It is not shown in Github.